package.json

```diff
@@ -57,6 +62,16 @@
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^4.1.2"
   },
+  "jest": {
+    "coverageThreshold": {
+      "global": {
+        "branches": 85,
+        "functions": 85,
+        "lines": 85,
+        "statements": -10
```

Comment:
Why is this -10?

Reply:
Statement -10 ensures that the test fails  if there are more than 10 uncovered statements

Reply:
Could you share the article for the same

Reply:
https://jestjs.io/docs/configuration#coveragethreshold-object

Reply:
It is bit confusing I feel, Could we go ahead with 95% here as well?

Reply:
This line can be removed then

Reply:
Do you feel removing the statements coverage altogether a good option or could we go ahead with 95% here as well

Reply:
Basically it is specified as a negative number to represent the maximum number of uncovered entities allowed.